eoverdracht/leveranciersspecificatie.md

```diff
@@ -254,15 +254,14 @@ Dit laatste endpoint dient het `[base]` path te zien waarop notificaties ontvang
 
 #### 4.1.3 Notificatie protocol
 
-Het voorstel is om het notificatie mechanisme compatible te houden met [FHIR subscriptions](http://hl7.org/fhir/STU3/subscription.html). Bij een volledige implementatie van FHIR subscriptions neemt het doelsysteem een abonnement op een FHIR search query bij een bronsysteem. Daarmee wordt expliciet vastgelegd wanneer een doelsysteem genotificeerd moet worden.
+Het FHIR STU3 notificatie en [subscriptions](http://hl7.org/fhir/STU3/subscription.html) model geeft onvoldoende ondersteuning voor deze bolt.
+Er is daarom gekozen om de specfieke Task identifier op te nemen in het notificatie pad.
+Dit wordt heroverwogen wanneer FHIR versie 5 gebruikt gaat worden.
+Het Task notificatie endpoint is specifiek voor deze bolt en zou niet voor andere doeleinden gebruikt moeten worden.
 
-Echter, wanneer een organisatie maar één generiek FHIR subscriptions endpoint kan registreren terwijl er verschillende systemen gebruikt worden voor de eOverdracht en andere Bolts of niet-Nuts processen, dan moeten subscriptions doorgestuurd worden naar verschillende applicaties, met alle complicaties van dien. Daarom is het voorstel om de specifieke notificatie endpoints voor verschillende use-cases apart te registreren en uit te wisselen.
+Wanneer er een task wordt toegevoegd zal het bronsysteem een notificatie \(lege POST volgens FHIR documentatie\) sturen naar het eerder geregistreerde endpoint van het doelsysteem. Dit is een signaal aan het doelsysteem om de FHIR task resource op te halen. Deze task kan opgehaald worden zonder geïdentificeerde gebruiker. Dit betekent dat de task resource geen persoonsgegevens bevat en dat eventuele referenties naar persoonsgegevens natuurlijk een geautoriseerde gebruiker vereisen.
```

Comment:
>Dit betekent dat de task resource geen persoonsgegevens bevat en dat eventuele referenties naar persoonsgegevens natuurlijk een geautoriseerde gebruiker vereisen.

Dit betekent dat de task resource geen persoonsgegevens (of referenties waarmee een persoon uniek geidentificeerd kan worden) mag bevatten.